Gemfile

```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 
 gem 'activerecord-import'
 gem 'activerecord-session_store'
-gem 'acts_as_list', github: 'swanandp/acts_as_list'
+gem 'acts_as_list', github: 'chrisortman/acts_as_list'
```

Comment:
Chris, I'm going to go ahead and close this one. Thanks for the work, we incorporated a lot of your suggestions into our test fixes, but we didn't see the need to update Acts as List for the test suite.